vcs/gitcmd/repo.go

```diff
@@ -343,6 +343,10 @@ func (r *Repository) commitLog(opt vcs.CommitsOptions) ([]*vcs.Commit, uint, err
 		args = append(args, "--skip="+strconv.FormatUint(uint64(opt.Skip), 10))
 	}
 
+	if opt.Path != "" {
+		args = append(args, "--follow")
+	}
+
 	// Range
 	rng := string(opt.Head)
```

Comment:
Right, I will do one of those two things (either check here, or document that caller is responsible for doing so), thanks!